README.md

```diff
@@ -1,3 +1,12 @@
+# [tedee Release Notes](https://tedee-com.github.io/tedee-release-notes/)
```

Comment:
Either remove this line (and change tedee to Tedee) or remove line number 3

Reply:
done